index.js

```diff
@@ -0,0 +1,159 @@
+const express = require("express");
+const fs = require("fs");
+const app = express();
+
+app.use(express.json());
+
+app.get("/person/:name", (req, res) => {
+  const idadeRandom = Math.floor(Math.random() * (80 - 18) + 18);
+
+  const jsonResponse = JSON.parse(`{
+        "person": {
+            "name": "${req.params.name}",
+            "idade": "${idadeRandom}"
+        } 
+    }`);
+  res.send(jsonResponse);
+});
+
+let persons = [];
+
+(() => {
+  fs.readFile('./persons.txt', (err, data) => {
+    persons = JSON.parse(data)
+    if(err) {
+      console.log('Cannot read file')
+      console.log(err)
+    }
+  })
+})()
+
+function setPersons() {
```

Comment:
Maybe a name line "savePeople()" would be better descriptive?

Reply:
Great suggestion!